tests/e2e-ui-tests/.eslintrc

```diff
@@ -0,0 +1,43 @@
+{
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "protractor": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    "array-callback-return": "warn",
+    "brace-style": ["error", "1tbs"],
+    "complexity": ["warn", 20],
+    "eqeqeq": "error",
+    "guard-for-in": "error",
+    "indent": ["error", 2],
+    "linebreak-style": ["error", "unix"],
+    "no-array-constructor": "error",
+    "no-console": "warn",
+    "no-lonely-if": "warn",
+    "no-loop-func": "warn",
+    "no-mixed-spaces-and-tabs": ["error"],
+    "no-nested-ternary": "error",
+    "no-spaced-func": "error",
+    "no-trailing-spaces": "error",
+    "semi": ["error", "always"],
+    "space-before-blocks": "error",
+    "space-before-function-paren": ["error", "never"],
+    "keyword-spacing": ["error"],
+    "curly": ["error", "all"],
+    "no-else-return": ["error"]
+  },
+  "globals": {
+    "angular": false,
+    "$fh": false,
+    "localStorage": false,
+    "window": false,
+    "expect": false,
+    "step": false
+  }
+}
```

Comment:
We should avoid having independent eslint files into the modules, let's try to inherit from the root and add the extra variables these files need (i.e. `env.protractor`, globals, relevant rules, etc):

[root eslintrc](https://github.com/feedhenry-raincatcher/raincatcher-angularjs/blob/master/package.json#L7)
[eslint docs](https://eslint.org/docs/user-guide/configuring#using-a-configuration-file)